main.go

```diff
@@ -335,10 +337,22 @@ func main() {
 		SelectorReconciler:     &nsSelReconciler,
 		EnableMetrics:          opts.enableMetrics,
 	}
+
+	OpReconciler := controllers.OperatorPolicyReconciler{
+		Client:   mgr.GetClient(),
+		Scheme:   mgr.GetScheme(),
+		Recorder: mgr.GetEventRecorderFor(controllers.ControllerName),
+	}
+
 	if err = reconciler.SetupWithManager(mgr); err != nil {
 		log.Error(err, "Unable to create controller", "controller", "ConfigurationPolicy")
 		os.Exit(1)
 	}
+
+	if err = OpReconciler.SetupWithManager(mgr); err != nil {
+		log.Error(err, "Unable to create controller", "controller", "OperatorPolicy")
+		os.Exit(1)
+	}
```

Comment:
Can we put this behind a feature flag? Something that would default to false in `parseOpts`. Then when we want to test stuff with it, we can turn it on by editing the deployment or something.

Reply:
I'll look into this.